hubblestack/extmods/modules/nebula_osquery.py

```diff
@@ -820,7 +823,7 @@ def _mask_object(object_to_be_masked, topfile):
                         # This means data is generated by osquery interactive shell
                         for query_name, query_ret in r.iteritems():
                             for query_result in query_ret['data']:
-                                if custom_mask_column and custom_mask_column in query_result:
+                                if enable_local_masking == 'true' and custom_mask_column and custom_mask_column in query_result:
```

Comment:
As with the comments above, let's use real `True` here.

```
if enable_local_masking is True
```

The rest of the instances should also be changed. We should never use `'true'`, it's much slower.

Reply:
Changes done and tested.